src/styles/Navigation.css

```diff
@@ -1,21 +1,45 @@
 .nav-bar {
-  height: 80px;
+  height: 100px;
   width: 100%;
   display: flex;
-  gap: 30px;
+  gap: 40px;
   align-items: center;
-  padding: 0 50px;
-  margin-bottom: 20px;
+  padding: 0 70px;
+  margin-bottom: 50px;
+  background-color: #fff;
 }
 
```

Comment:
- [ ] Good job for aligning the items to the center 👏 However, I noticed that now the title/logo/brand name is misaligned, kindly make sure to fix this so that it appears on one line according to the design guidelines we have. Please go through the following images for clarity 👇 Kindly rectify this as this was not present in the last review.

Design before | Design now | Given design
---| ---| ---|
![image](https://github.com/Bhekisile/Bookstore/assets/106119105/0a6f4c4b-fc94-42d9-b962-a8585913b8f2) | ![image](https://github.com/Bhekisile/Bookstore/assets/106119105/f852fb52-b135-4a11-b929-16bc0c1ea54a) | ![image](https://github.com/Bhekisile/Bookstore/assets/106119105/3548e96b-f9a9-465c-8efc-869ac55f0019)